### PR TITLE
Admin: User list tweaks

### PR DESCRIPTION
**Pull request by Clarity-89**
**What this PR does / why we need it**:
 - Add radio group filter (all users / active last 30 days)
 - Add header to Server admin column (shield icon)
 - Rename Seen column header to Last active
 - Display relative time in `Last active` as full words (minute, hour, week, month, …) instead of abbreviations (m, h, d, …)

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana-enterprise/issues/1836

**Special notes for your reviewer**:
Screenshot 2021-08-31 at 14 53 07

### CODE REVIEW

**Review by hugohaggmark — APPROVED** [review]
Frontend code looks good and works as expected 🎉  I found some a11y related issues that would be great if you could improve. Thank you!

File: public/app/features/admin/UserListAdminPage.tsx [hunk 1]
```diff
@@ -122,6 +151,7 @@ const renderUser = (user: UserDTO) => {
           </a>
         )}
       </td>
+      <td className="link-td">{user.lastSeenAtAge && <a href={editUrl}>{user.lastSeenAtAge}</a>}</td>
```
[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/features/admin/UserListAdminPage.tsx:154]: Great suggestions! About link accessibility, were you thinking about adding a more descriptive `title` attribute to it, or any other solution?

[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/features/admin/UserListAdminPage.tsx:154]: > The server admin icon shouldn't be part of the tab order, I think.

Seems like that icon is used to link to the edit user view, so it'd be in the tab order. Probably need to add a description to make that clearer.

[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/features/admin/UserListAdminPage.tsx:154]: > About link accessibility, were you thinking about adding a more descriptive title attribute to it, or any other solution?

Nevermind, missed the part about `aria-label` 🙈

[the remark this answers is not in the document]
Comment (hugohaggmark) [reply public/app/features/admin/UserListAdminPage.tsx:154]: The difference between `aria-label` and `title` in this case is that it gets announced as

| aria-label | title |
--- | --- 
link, Last seen at 1 minute | link 1 minute, ....  Last seen at

Not sure which one is the best, if we use aria-label we need to add the value in the aria-label not a big fan of that. If we add title it's announced after the value. (Using VoiceOver on Mac)

[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/features/admin/UserListAdminPage.tsx:154]: The issue here is that the link directs to the user's edit view (that's a way of making whole table row act as a link), so the href and content of the link don't match. Maybe I'll try to combine them somehow in the aria-label 🤔

[the remark this answers is not in the document]
Comment (hugohaggmark) [reply public/app/features/admin/UserListAdminPage.tsx:154]: Yepp, I understand it's tricky

**Comment by hugohaggmark** [comment]
Works way better, I think login, email and name could be more descriptive too.

**Pull request merged**